packages/studio-plugin/src/createStudioStylingPlugin.ts

```diff
@@ -32,16 +36,18 @@ export default function createStudioStylingPlugin(): PluginOption {
   };
 }
 
-function isImportedByStudio(id: string) {
-  return !!(id.match(/.*@yext\/studio.*/) || id.match(/.*packages\/studio*/));
+function isImportedByStudio(importer: string) {
+  return !!(
+    importer.match(/.*@yext[\\\\/]studio.*/) || importer.startsWith(packagesDir)
```

Comment:
we can probably use `path.sep` to be more specific here instead of checking for either forward or back slashes. alternatively, we could run the `importer` path through `upath` to begin with and then we can be assured that the path separator with always be `/` so we wouldn't need special windows handling

Reply:
I went with the latter!